### PR TITLE
Q3 release post

### PR DESCRIPTION
**Pull request by EDDYMENS**

### CODE REVIEW

File: _blog/blogposts/blogposts_info.json [hunk 1]
```diff
@@ -13049,8 +13089,8 @@
     "filename": "use-a-green-screen-in-javascript-with-vonage-video"
   },
   {
-    "title": "Add Strong PSD2 Authentication to Your Application",
```
Comment (devwithzachary) [inline _blog/blogposts/blogposts_info.json:13052]: Why are we changing things in the blogpost info? Is this meant to have been committed?

File: _blog/blogposts/en/q3-2022-third-quarter-in-review.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+It's time to get rid of the old and bring in the new. We released major Ruby and Java SDK updates, including improvements, fixes, and deprecations.
+
+
+You can find details on what's changed in the [Ruby v3](https://developer.vonage.com/blog/22/08/08/announcing-python-sdk-version-v3) and [Java v7](https://developer.vonage.com/blog/22/08/22/announcing-the-vonage-java-sdk-v7-0-0) release posts.
+
+**The V-app**
+
+Do you know what is better than SDKs? Demo apps showing SDKs in action. Let me introduce you to the [V-app](https://github.com/nexmo-community/clientsdk-the-v-app) a multi-platform (iOS, Android, Web) demo app that demonstrates the implementation of voice and messaging funtionalities.
+
+Here is [a post](https://developer.vonage.com/blog/22/08/25/introducing-the-vonage-client-sdk-v-app-demo-projects
```
Comment (devwithzachary) [inline _blog/blogposts/en/q3-2022-third-quarter-in-review.md:70]: Worth crediting Abdul here for the post (As done with the other posts above)

**Review by devwithzachary — COMMENTED** [review]
Complete of points in line, otherwise looks good!

File: _blog/blogposts/blogposts_info.json (hunk 1 above)
Comment (EDDYMENS) [reply _blog/blogposts/blogposts_info.json]: I autogenerated on ADP during the authoring process, its undone now :)

File: _blog/blogposts/en/q3-2022-third-quarter-in-review.md (hunk 2 above)
Comment (EDDYMENS) [reply _blog/blogposts/en/q3-2022-third-quarter-in-review.md:67]: Abdul got his cake now :)

**Review by devwithzachary — APPROVED** [review]

**Pull request merged**